FortranCon2020-community/README.md

```diff
@@ -4,4 +4,23 @@
 
 **Title**: Toward a thriving open source Fortran community
 
-**Abstract**: TODO
+**Abstract**:
+A thriving community around a programming language is essential for onboarding
+newcomers to the language and retaining existing users.
+Besides a few discussion boards and mailing lists, Fortran has not had a 
+healthy online community in the modern internet era, like many other 
+programming languages have.
+Despite the developments of the language itself, its user base has been 
+declining as a result. 
+Many new software projects are now being started in languages like C++, Python,
+or Julia, despite Fortran being better suited for the task at hand.
+In this talk, we'll discuss the need for and the ongoing development of the new
+fortran-lang (https://fortran-lang.org) community and its suite of projects.
+Fortran-lang aims to provide a central place for Fortran users, beginning and 
+expert alike, to find a set of recommended tools, libraries, tutorials, and 
+discussion venues.
+We believe that an organized online community, besides the ongoing work by the
+Standards Committee and compiler developers, is essential for the long-term 
+thriving--and not just mere survival--of Fortran.
```

Comment:
```suggestion
thriving -- and not just mere survival -- of Fortran.
```

Reply:
Most style guides recommend no spaces around em dash, unless writing for a newspaper. I'll keep no spaces.